source/_docs/configuration/troubleshooting.markdown

```diff
@@ -20,7 +20,7 @@ If you have incorrect entries in your configuration files you can use the config
 One of the most common problems with Home Assistant is an invalid `configuration.yaml` or other configuration file.
 
 - With Home Assistant OS and Supervised you can use the [`ha` command](/hassio/commandline/#home-assistant): `ha core check`.
-  - You can test your configuration with Home Assistant Core using the command line with: `hass --script check_config`. If you need to provide the path for your configuration you can do this using the `-c` argument like this: `hass --script check_config -c /path/to/your/config/dir`.
+  - You can test your configuration with Home Assistant Core using the command line with: `ha core check`. If you need to provide the path for your configuration you can do this using the `--config` argument like this: `ha core check --config /path/to/your/config/dir`.
```

Comment:
This bullet point is about running Home Assistant Core. `hass` was correct, this change needs to be reverted.